partiql-lang/src/test/kotlin/org/partiql/lang/eval/IonAnnotationTests.kt

```diff
@@ -0,0 +1,57 @@
+package org.partiql.lang.eval
+
+import com.amazon.ion.IonSystem
+import com.amazon.ion.system.IonSystemBuilder
+import org.junit.Test
+import org.partiql.lang.CompilerPipeline
+
+/**
+ * The following tests demonstrate the current IonValue annotation behavior and tracks possible regressions in future commits.
+ * This behavior is what the Kotlin implementation defines and is subject to change pending resolution of
+ * https://github.com/partiql/partiql-spec/issues/63.
+ */
+class IonAnnotationTests : EvaluatorTestBase() {
+    // Round-tripped Ion value with Annotation (IonValue -> ExprValue -> IonValue) results in the elided annotation.
+    @Test
+    fun ionValueWithAnnotationExprValueRoundTrip() {
+        val ion: IonSystem = IonSystemBuilder.standard().build()
+        val ionValue = ion.singleValue("1")
+        val exprValue = ExprValue.of(ion.singleValue("annotation::1"))
+        val roundTripped = exprValue.toIonValue(ion)
+        assertEquals(ionValue, roundTripped)
```

Comment:
Nit: but I would do 
```suggestion
        val inputIonValue = ion.singleValue("annotation::1")
        val exprValue = ExprValue.of(inputIonValue)
        val roundTripped = exprValue.toIonValue(ion)
        assertEquals(ion.singleValue("1"), roundTripped)
```

just so it is more clear that the ion annotation get dropped.